core/src/main/java/com/linecorp/armeria/internal/common/util/SslContextUtil.java

```diff
@@ -100,10 +101,23 @@ public static SslContext createSslContext(
             boolean tlsAllowUnsafeCiphers,
             Iterable<? extends Consumer<? super SslContextBuilder>> userCustomizers,
             @Nullable List<X509Certificate> keyCertChainCaptor) {
+        return createSslContext(builderSupplier, forceHttp1, Flags.tlsEngineType(), tlsAllowUnsafeCiphers,
```

Comment:
We don't have to keep backward compatibility for internal APIs. Let's remove this method.